core/README.md

```diff
@@ -7,14 +7,6 @@
   - [Extensions](#extensions)
   - [Structuring Catalog Hierarchies](#structuring-catalog-hierarchies)
 
-- **OpenAPI specification:** [openapi.yaml](openapi.yaml) ([rendered version](https://api.stacspec.org/v1.0.0-rc.2/core)),
```

Comment:
Is this intentional? If yes, why?

Reply:
It's not -- it's a problem with the VS Code markdown plugin. I'll fix this.